addons/point_of_sale/static/src/app/screens/product_screen/order_summary/order_summary.js

```diff
@@ -209,7 +205,7 @@ export class OrderSummary extends Component {
                 selectedLine.set_quantity(newQuantity, true);
             } else {
                 newLine.set_quantity(-decreasedQuantity, true);
-                order.add_orderline(newLine);
+                this.currentOrder.add_orderline(newLine);
```

Comment:
`add_orderline` do not exist anymore in this version, can you also check `_showDecreaseQuantityPopup` ?